Example/ConsentViewController_ExampleTests/Helpers/SourcePointClientMock.swift

```diff
@@ -165,7 +165,8 @@ class SourcePointClientMock: SourcePointProtocol {
                 legIntCategories: nil,
                 legIntVendors: nil,
                 vendors: nil,
-                categories: nil
+                categories: nil,
+                specialFeatures: nil
```

Comment:
for the future references, can you add `,` at the end of this line, so in the future if you add something new to the constructor of `GDPRChoiceResponse`, the GH checks will not mark it as an update, only as an addition?

Reply:
@bohdan-go-wombat the linter complains about dangling commas :)

Reply:
@bohdan-go-wombat Yeah, the swift language does not support commas after the last parameter of the declaration